eng/scripts/Language-Settings.ps1

```diff
@@ -521,3 +521,10 @@ function UpdateDocsMsPackages($DocConfigFile, $Mode, $DocsMetadata) {
   }
   Set-Content -Path $DocConfigFile -Value $outputLines
 }
+
+function Import-Dev-Cert-net
```

Comment:
Who consumes this?

Reply:
https://github.com/Azure/azure-sdk-tools/blob/c0d142ae047f0d561a31f9989c63896cc5ffc70e/eng/common/scripts/common.ps1#L47